### PR TITLE
button dropdown and plus icon

### LINKED ISSUE #19: Add plus icon and button dropdown

### PR DESCRIPTION
**Pull request by pawan-blotout**
Resolves #19

### CODE REVIEW

**Review by NejcZdovc — COMMENTED** [review]
please create storybook for it

File: src/icons/plus.jsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export default ({ width, height }) => {
```
Comment (NejcZdovc) [inline src/icons/plus.jsx:1]: we have Add icon that you can use

Comment (pawan-blotout) [reply src/icons/plus.jsx:1]: I have used Add icon.
Should I remove Play icon ?

Comment (NejcZdovc) [reply src/icons/plus.jsx:1]: yes please remove Plus icon

Comment (pawan-blotout) [reply src/icons/plus.jsx]: Done

File: src/typings/index.d.ts [hunk 2]
```diff
@@ -98,6 +98,23 @@ interface DropdownProps {
   sideHeading?: string
 }
 
+interface ButtonDropdownProps {
+  handleChange: (value: SelectValue) => void
+  value: SelectValue
```
Comment (NejcZdovc) [inline src/typings/index.d.ts:103]: this should be `value?: SelectValue | null`

Comment (pawan-blotout) [reply src/typings/index.d.ts]: Done

**Review by NejcZdovc — APPROVED** [review]

**Pull request merged**